lib/src/common/parameters/excluded_identifiers_list_parameter.dart

```diff
@@ -26,6 +26,39 @@ class ExcludedIdentifiersListParameter {
     for (final item in excludeList) {
       if (item is Map) {
         exclude.add(ExcludedIdentifierParameter.fromJson(item));
+      } else if (item is String) {
+        exclude.add(
+          ExcludedIdentifierParameter(
+            declarationName: item,
+          ),
+        );
+      }
+    }
+    return ExcludedIdentifiersListParameter(
+      exclude: exclude,
+    );
+  }
+
+  /// Method for creating from json data with default params
+  factory ExcludedIdentifiersListParameter.defaultFromJson(
+    Map<String, dynamic> json,
+  ) {
+    final exclude = <ExcludedIdentifierParameter>[];
+
+    final excludeList =
+        json[ExcludedIdentifiersListParameter.excludeParameterName]
+                as Iterable? ??
+            [];
+
+    for (final item in excludeList) {
+      if (item is Map) {
+        exclude.add(ExcludedIdentifierParameter.fromJson(item));
+      } else if (item is String) {
+        exclude.add(
+          ExcludedIdentifierParameter(
+            declarationName: item,
+          ),
+        );
```

Comment:
You can replace all this with a call to another factory, to avoid duplication
```dart
  return ExcludedIdentifiersListParameter.fromJson(excludeList: excludeList);
```